### ISSUE TITLE
leaderboard v2 - add autoupdate

### ISSUE THREAD

**Issue by KennethEnevoldsen** [embeddings-benchmark/mteb#1605]
Seems like we currently need to manually update the MTEB leaderboard (@orionw I believe you fixed the old one. I believe we really only need to restart the space daily)

**Comment by orionw** [comment]
Yes, definitely! I'm trying to figure out how it get's it data, seems like there's a function in MTEB now that provides the results, is that right? (also cc'ing @x-tabdeveloping). The way I restarted it daily before was by updating the results file in the repo daily and then the results repo would re-start.

If I understand correctly, would the best way to do this be to auto-restart the leaderboard on MTEB merges? Or perhaps MTEB results github merges? I don't see the connection to the `results` github, but I haven't traced it all the way back yet.

**Comment by orionw** [comment]
Totally works great :) Do you mind pointing me to that step? I can work out how to get the leaderboard to refresh after it.

**Comment by orionw** [comment]
@x-tabdeveloping if you have any pointers :)
